dashboard/app/reporting_email.go

```diff
@@ -552,4 +570,30 @@ func appURL(c context.Context) string {
 	return fmt.Sprintf("https://%v.appspot.com", appengine.AppID(c))
 }
 
+func separateCc(to, cc []string) []string {
+	mb := make(map[string]bool, len(to))
+	newCc := []string{}
+	for _, x := range to {
```

Comment:
`email` instead of `x`?

Reply:
`email` generated building problems on ci/smoke so I used `addr` instead.